framework/src/utils/DelimitedFileReader.C

```diff
@@ -207,5 +213,190 @@ DelimitedFileReader::readData(std::ifstream & stream_data, std::vector<double> &
     // Append data
     output.insert(output.end(), row.begin(), row.end());
   }
+
+  // If the names have not been assigned, create the default names
+  if (_names.empty())
+  {
+    _names.resize(n_cols);
+    int padding = std::log(n_cols);
```

Comment:
OK - this is borderline clever. I know what you are doing because I've written this kind of code. It's worth documenting how it works. However there are a few problems. First, you are using the wrong `log`. You need `log10`. Second, wrong type. There are no `int` return overloads so you are truncating here. I'd suggest an explicit cast if you are indeed intending to truncate. This might even be a compile warning as it sits.

Reply:
I will fix.